plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -176,11 +192,26 @@ public function track_product_published( $post_id, $post, $update, $post_before
 		$product = wc_get_product( $post_id );
 
 		$properties = array(
-			'product_id'      => $post_id,
-			'product_type'    => $product->get_type(),
+			'attributes' => count( $product->get_attributes() ),
+			'categories' => count( $product->get_category_ids() ),
+			'cross-sells' => $product->get_cross_sell_ids() ? 'yes' : 'no',
+			'description' => $product->get_description() ? 'yes' : 'no',
+			'dimensions' => $product->get_dimensions() ? 'yes' : 'no',
```

Comment:
I think this one also returns an array regardless of whether or not fields have been filled in.

```
		return array(
			'length' => $this->get_length(),
			'width'  => $this->get_width(),
			'height' => $this->get_height(),
		);
```